peachjam/models/journals_books.py

```diff
@@ -0,0 +1,21 @@
+from django.db import models
+
+from peachjam.models import CoreDocument
+
+
+class Book(CoreDocument):
+    publisher = models.CharField(max_length=2048)
+
+    def save(self, *args, **kwargs):
+        self.frbr_uri_doctype = "book"
```

Comment:
This will need to be `doc` and the subtype `book`, similarly for the Journal type. There are a limited set of [official FRBR doctypes](http://docs.oasis-open.org/legaldocml/akn-core/v1.0/os/part1-vocabulary/akn-core-v1.0-os-part1-vocabulary.html#_Toc523925025) which we must obey, but the subtype allows anything.